### PR TITLE
feat(fscomponents): add completed state accessibility label for Step

### PR DESCRIPTION
**Pull request by chukohsin**
This changes add completed state accessibility label for Step component.

### CODE REVIEW

File: packages/fscomponents/src/components/Step.tsx [hunk 1]
```diff
@@ -53,6 +53,9 @@ export class Step extends Component<StepProps> {
         <TouchableOpacity
           onPress={this.props.onPress}
           style={[styles.container, this.props.style]}
+          accessibilityTraits='button'
+          accessibilityComponentType='button'
+          accessibilityLabel={`${this.props.title}, Step Completed`}
```
Comment (justin-endler) [inline packages/fscomponents/src/components/Step.tsx]: @KoHsinChuBB "Step Completed" should be added to the `fsi18n` package and pulled from there.

File: packages/fscomponents/src/components/Step.tsx [hunk 2]
```diff
@@ -53,6 +53,9 @@ export class Step extends Component<StepProps> {
         <TouchableOpacity
           onPress={this.props.onPress}
           style={[styles.container, this.props.style]}
+          accessibilityTraits='button'
```
Comment (justin-endler) [inline packages/fscomponents/src/components/Step.tsx]: `accessibilityTraits` and `accessibilityComponentType` are deprecated for iOS and Android respectively. Can `accessibilityRole` be used instead?

File: packages/fscomponents/src/components/Step.tsx (hunk 1 above)
Comment (chukohsin) [reply packages/fscomponents/src/components/Step.tsx]: @justin-endler  Will fix it, thanks.

File: packages/fscomponents/src/components/Step.tsx (hunk 2 above)
Comment (chukohsin) [reply packages/fscomponents/src/components/Step.tsx]: Yes, we can use it but there will be typescript error if that's okay.

**Pull request merged**